pkg/parser/ast/ddl.go

```diff
@@ -2176,6 +2176,7 @@ type ResourceGroupOption struct {
 	StrValue          string
 	UintValue         uint64
 	BoolValue         bool
+	RUUnlimited       bool // mark `RU_PER_SEC` is `unlimited`
```

Comment:
What about share BoolValue?

Reply:
Okay, It does work, the fields look like all mixed up